tests/Integration/Pagination/PaginateDirectiveDBTest.php

```diff
@@ -88,6 +89,51 @@ public function testSpecifyCustomBuilder(): void
         ]);
     }
 
+    public function testSpecifyCustomBuilderForRelation(): void
+    {
+        $post = factory(Post::class)->create();
+
+        $this->schema = /** @lang GraphQL */ '
+        type Post {
+            id: ID!
+        }
+
+        type User {
+            id: ID!
+            posts: [Post!]! @paginate(builder: "' . $this->qualifyTestResolver('builderForRelation') . '")
+        }
+
+        type Query {
+            user(id: ID!): User @find
+        }
+        ';
+
+        // The custom builder is supposed to change the sort order
```

Comment:
I know this was not the purpose of adding the test, but it would still be great if the result would prove that the builder works.

Perhaps just add 2 posts, query for 2 but let the builder exclude one of them by some condition.

Reply:
Ah chips, also missed this comment I copied and pasted, oeps.

I've fixed it to do what the comment said (reversing the order).